### PR TITLE
Use python36 and install tox to the right path

### PR DESCRIPTION
**Pull request by sayf-eddine-scality**
Fixes: #940

### CODE REVIEW

File: eve/main.yml [hunk 1]
```diff
@@ -150,7 +150,7 @@ stages:
           env:
             ISO_MOUNTPOINT: >
               /srv/scality/metalk8s-%(prop:metalk8s_short_version)s
-          command: tox -e tests-local
+          command: /usr/local/bin/tox -e tests-local
```
Comment (gdemonet) [inline eve/main.yml]: No, we might install `tox` only for the `eve` user, which would land in `/home/eve/.local/bin`

File: eve/workers/openstack-multiple-nodes/requirements.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ yum install -y epel-release
 PACKAGES=(
     curl
     git
-    python34-pip
+    python36-pip
```
Comment (gdemonet) [inline eve/workers/openstack-multiple-nodes/requirements.sh:8]: We also need to replace `pip3` with `pip34` below

**Review by gdemonet — DISMISSED** [review]

File: eve/workers/openstack-multiple-nodes/requirements.sh (hunk 2 above)
Comment (sayf-eddine-scality) [reply eve/workers/openstack-multiple-nodes/requirements.sh:8]: pip34 or pip36 ?

Comment (gdemonet) [reply eve/workers/openstack-multiple-nodes/requirements.sh:8]: Actually, `pip3.6` (oops)

**Comment by sayf-eddine-scality** [comment]
@bert-e reset

**Comment by sayf-eddine-scality** [comment]
@bert-e reset

**Comment by sayf-eddine-scality** [comment]
@bert-e approve

**Review by TeddyAndrieux — APPROVED** [review]
Green build lets approve :)

**Pull request merged**